submissions/YuliiaDyka/pop-up/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Popup</title>
+</head>
+    <body>
+        <header class="header">
+            <div class="logo">
+                <a class="logo__link" href="https://github.com/Yuliiadd/kottans-frontend"><img class="logo__img" src="img/logo.png" alt="Go to homepage"></a>
+            </div>
+            <nav class="nav">
```

Comment:
Please fix the mobile version. [See video](https://share.vidyard.com/watch/4nSJu5FTQwfv9MVeiMM4qE?)